# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/*/libjxl/system-highway.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- libjxl-0.5/deps.sh.vanilla	2021-09-02 20:07:39.881273528 +0200
+++ libjxl-0.5/deps.sh	2021-09-02 20:07:46.007273284 +0200
@@ -69,7 +69,7 @@
   fi
 
   # Sources downloaded from a tarball.
-  download_github third_party/highway google/highway
+  #download_github third_party/highway google/highway
   download_github third_party/lodepng lvandeve/lodepng
   download_github third_party/sjpeg webmproject/sjpeg
   download_github third_party/skcms \
